components/log-viewer-webui/server/src/plugins/MongoSocketIoServer/index.ts

```diff
@@ -288,7 +292,7 @@ class MongoSocketIoServer {
     #unsubscribe (socket: MongoCustomSocket, queryId: number) {
         const queryHash: string | undefined = this.#queryIdToQueryHashMap.get(queryId);
         if ("undefined" === typeof queryHash) {
-            this.#fastify.log.error(`QueryId ${queryId} not found in query map`);
+            this.#fastify.log.error(`query:${queryId} not found in query map`);
```

Comment:
for consistency with the above?
```suggestion
            this.#fastify.log.error(`Query:${queryId} not found in query map`);
```